aw_watcher_afk/afk.py

```diff
@@ -26,40 +26,43 @@
 
 class Settings:
     def __init__(self, config_section):
+        # Time without input before we're considering the user as AFK
         self.timeout = config_section.getfloat("timeout")
-        self.update_time = config_section.getfloat("update_time")
```

Comment:
Now when we have client-side heartbeat merging we can remove update_time altogether in aw-watcher-afk.

Reply:
Done!